src/Symfony/Component/Console/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ---
 
  * Add support for `FORCE_COLOR` environment variable
+ * Add 'verbosity' argument to 'mustRun' process helper method
```

Comment:
Please use ticks instead of single quotes, thanks

Reply:
@OskarStark done.